### ISSUE TITLE
[Bug]: Accessibility: Table is missing list items

### ISSUE THREAD

**Issue by mpivchev** [nextcloud/server#59339]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

<img width="1079" height="163" alt="Image" src="https://github.com/user-attachments/assets/893fdac7-e95b-42c5-a6ab-65542f6d2c23" />

### Steps to reproduce

1. Go to Security
2. Observe

### Expected behavior

Table should have list items

### Nextcloud Server version

32

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```json

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell

```

### Nextcloud Logs

```json

```

**Comment by susnux** [comment]
Technical background: https://www.w3.org/WAI/standards-guidelines/act/rules/bc4a75/proposed/
So this is basically for WCAG 2 rule 1.3.1: https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=131#info-and-relationships
